pkg/utils/utils.go

```diff
@@ -388,3 +385,24 @@ func retrieveNSList(nsList map[string]bool) []string {
 	}
 	return namespaces
 }
+
+// This utility returns a true/false depending on whether
+// the user requires advanced L4 functionality
+func GetAdvancedL4() bool {
+	advanceL4 := os.Getenv(ADVANCED_L4)
+	if advanceL4 == "true" {
```

Comment:
A better way of expressing this is:
	if ok, _ := strconv.ParseBool(os.Getenv(ADVANCED_L4)); ok {
		return true
	}

Reply:
Updated.